android/app/src/main/java/app/organicmaps/bookmarks/BookmarkCategoriesFragment.java

```diff
@@ -177,7 +178,11 @@ public ArrayList<MenuBottomSheetItem> getMenuBottomSheetItems(String id)
       items.add(new MenuBottomSheetItem(
           R.string.export_file,
           R.drawable.ic_share,
-          () -> onShareActionSelected(mSelectedCategory)));
+          () -> onShareActionSelected(mSelectedCategory, KmlFileType.Text)));
+      items.add(new MenuBottomSheetItem(
+              R.string.export_file_gpx,
```

Comment:
The indentation doesn't match the code above.

Reply:
Fixed